src/query/Index.hpp

```diff
@@ -72,6 +72,9 @@ class Index {
   /// Get n number of closest vertices to the given vertex
   VertexMatch getClosestVertex(const Eigen::VectorXd &sourceCoord);
 
+  /// Get n number of closest vertices to the given vertex
+  std::vector<VertexID> getClosestVertices(const mesh::Vertex &centerVertex, unsigned int n);
+
```

Comment:
Seems like the documentation in line 72 is wrong?
Why `Eigen::VectorXd` vs `mesh::Vertex`?

Reply:
> Seems like the documentation in line 72 is wrong?

Yep

> Why Eigen::VectorXd vs mesh::Vertex?

@davidscn True. We should keep the interface consistent. It's also way easier to test.